thorlcr/activities/indexread/thindexreadslave.cpp

```diff
@@ -322,7 +330,17 @@ class CIndexReadSlaveBase : public CSlaveActivity
             currentManager = keyManager;
         }
     }
-    virtual bool keyed() { return false; }
+    virtual bool incKeyedExceedsLimit()
+    {
+        if (keyedLimit == RCMAX)
```

Comment:
not sure if this test is worth it, but not a problem if it is.  (Depends if you want to output keyedProcessed as a stat)

Reply:
you're right, I've removed it.